extensions/lesion-tracker/src/LTStudyBrowser/LTStudyBrowser.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+import { RoundedButtonGroup } from '@ohif/ui';
+
+import mockData from './mockData.js';
+import { AllStudies } from './AllStudies.js';
+import { ViewingStudies } from './ViewingStudies.js';
+
+import './LTStudyBrowser.styl';
+
+const findStudy = (studies, studyInstanceUid) => {
+  return studies.find(study => study.studyInstanceUid === studyInstanceUid);
+};
+
+const filterStudies = (studies, filteredStudiesUids = []) => {
+  return studies.filter(
+    study => !filteredStudiesUids.includes(study.studyInstanceUid)
+  );
+};
+
+const tabs = [
+  {
+    name: 'Viewing',
```

Comment:
These tab names are definitive? I'm not sure if it would be necessary to add `i18n` here.

Reply:
Naming is not final yet I think.